### PR TITLE
Add losses and minor fixes

### PR DESCRIPTION
**Pull request by KrzysiekDD**
Add content, style, tv losses, update gram matrix functionality and flake8 warnings

### CODE REVIEW

File: nst/losses/style_loss.py [hunk 1]
```diff
@@ -1,36 +1,44 @@
 import torch
 from typing import List
+from torch.nn.functional import mse_loss
 
 
+# style_target_features, image_features are lists of tensor containing respective feature maps
 class StyleLoss:
-    def __init__(self, style_target_features: List[torch.Tensor]):
+    def __init__(self, style_target_features: List[torch.Tensor]):  # Gram matrix operation on target style image
```
Comment (Walusus) [inline nst/losses/style_loss.py:8]: I think this comment is redundant.

File: nst/losses/style_loss.py [hunk 2]
```diff
@@ -1,36 +1,44 @@
...
         :param style_target_features: list of style target feature maps
         """
-        # TODO compute gram matrix for each style target feature map
-        pass
+        self.style_gram_matrix_list = []
+        for feature_map in style_target_features:
+            self.style_gram_matrix_list.append(StyleLoss._gram_matrix(feature_map))
 
-    def __call__(self, image_features: List[torch.Tensor]) -> torch.Tensor:
+    def __call__(self, image_features: List[torch.Tensor]) -> torch.Tensor:  # Style loss for specific iteration
```
Comment (Walusus) [inline nst/losses/style_loss.py:19]: Same as above :)

File: nst/losses/style_loss.py [hunk 3]
```diff
@@ -1,36 +1,44 @@
...
-        # TODO compute loss based on input and target gram matrices
-        pass
+        sum_of_losses = torch.zeros(image_features[0].shape[0])
+        for index, feature_map in enumerate(image_features):
+            feature_map_gram_matrix = StyleLoss._gram_matrix(feature_map)
+            sum_of_losses += mse_loss(self.style_gram_matrix_list[index], feature_map_gram_matrix)
+        return sum_of_losses
 
-    @staticmethod
+    @staticmethod  # Gram matrix operation on the input
```
Comment (Walusus) [inline nst/losses/style_loss.py:33]: Same :)

**Review by Walusus — CHANGES_REQUESTED** [review]
The code looks good, only fix comments.

**Comment by KrzysiekDD** [comment]
Adressed review suggestions

**Review by Walusus — APPROVED** [review]
Ready to go :100:

**Pull request merged**